info/src/main/java/edu/artic/info/Loading.kt

```diff
@@ -0,0 +1,11 @@
+package edu.artic.info
+
+/**
+ * @author Sameer Dhakal (Fuzz)
+ */
+sealed class LoadStatus {
+    class Loading : LoadStatus()
+    class Error(val error: Throwable) : LoadStatus()
+    class None : LoadStatus()
```

Comment:
Replace `class` with `object`, rename from `None` to `Inactive`. `None` seems to me to be too similar to meaning "no content available" instead of "not currently loading" to be instantly clear.

Reply:
None as there is no load status right now. it is the default area, unsure if it actually needs to be replaced.

Reply:
By that logic shouldn't `Loading` be `Present` because there _is_ a load?